static/locales/en-US.yaml

```
@@ -1217,3 +1217,5 @@ KeyboardShortcutPrompt:
   Skip by Tenths: Skip through video by percentage (3 skips to 30% of duration)
   Home: Seek to the beginning of the video
   End: Seek to the end of the video
+  Skip to Next Video: Skip to the next video in a playlist or next recommended video if any
+  Skip to Previous Video: Skip to the previous video (when using a playlist)
```

Just to be consistent (I will apply this myself
```suggestion
  Skip to Previous Video: Skip to the previous video in a playlist
```